src/scripts/main.js

```diff
@@ -1,3 +1,335 @@
+          order = true;
+        }
+
+        if (order) {
+          rows.sort((a, b) => +a.children[3].textContent
+            - +b.children[3].textContent);
+        } else {
+          rows.sort((a, b) => +b.children[3].textContent
+            - +a.children[3].textContent);
+        }
+        lastSorted = 'Age';
+        order = !order;
+        break;
+
+      case 'Salary':
+        if (lastSorted !== 'Salary') {
+          order = true;
+        }
+
+        if (order) {
+          rows.sort((a, b) => convertSalary(a.children[4].textContent)
+            - convertSalary(b.children[4].textContent));
+        } else {
+          rows.sort((a, b) => convertSalary(b.children[4].textContent)
+            - convertSalary(a.children[4].textContent));
+        }
+        lastSorted = 'Salary';
+        order = !order;
+    };
+
+    for (const row of rows) {
+      tableBody.append(row);
+    }
+  };
+
+  if (e.target.closest('tbody')) {
+    rows.forEach((row) => {
+      row.classList.remove('active');
+    });
+
+    e.target.closest('tr').classList.add('active');
+  }
+});
+
+const body = document.querySelector('body');
+const newEmployeeForm = document.createElement('form');
+
+newEmployeeForm.classList.add('new-employee-form');
+
+newEmployeeForm.insertAdjacentHTML('afterbegin', `
+    <label>
+      Name: 
+      <input 
+        name="name" 
+        type="text"
+        data-qa="name" 
+        required
+      >
+    </label>
+    <label> 
+      Position: 
+      <input 
+        name="position" 
+        type="text" 
+        data-qa="position" 
+        required
+      >
+    </label>
+    <label> 
+      Office: 
+      <select 
+        name="office" 
+        type="text" 
+        data-qa="office" 
+        required
+      >
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <label> 
+      Age: 
+      <input 
+        name="age" 
+        type="number" 
+        min="0" 
+        data-qa="age" 
+        required
+      >
+    </label>
+    <label>
+      Salary: 
+      <input 
+        name="salary"
+        type="number" 
+        min="0" 
+        data-qa="salary" 
+        required
+      >
+    </label>
+
+    <button class="button"> Save to table </button>
+`);
+
+body.append(newEmployeeForm);
+
+function showNotification(type, title, message) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.classList.add('notification', type);
+
+  notification.insertAdjacentHTML('afterbegin', `
+    <h2 class="title">${title}</h2>
+    <p>${message}</p>
+  `);
+
+  body.append(notification);
+
+  setTimeout(() => {
+    notification.remove();
+  }, '3000');
```

Comment:
You can place such a code in single line:
```
setTimeout(() => notification.remove(), 3000);
```
But it's not critical. If you like your syntax, it's ok.
And it's better to use a `number` type for the timeout.